tests/dflowfm/test_mdu.py

```diff
@@ -472,3 +472,54 @@ def test_mdu_unknown_keywords_loading_thrown_valueerror_for_unknown_keyword_does
             assert len(excluded_fields) > 0
             for excluded_field in excluded_fields:
                 assert excluded_field not in str(exc_err.value)
+
+    def test_mdu_datetime_yyyymmdd_format(self, tmp_path):
+        tmp_mdu = """
+        [General]
+        fileVersion           = 1.09
+        fileType              = modelDef
+        program               = D-Flow FM
+        version               = 1.2.100.66357
+        autoStart             = 0
+        pathsRelativeToParent = 0
+
+        [Time]
+        startDateTime         = 20230101
+        stopDateTime          = 20230102
+        """
+
+        tmp_mdu_path = tmp_path / "tmp.mdu"
+        tmp_mdu_path.write_text(tmp_mdu)
+
+        model = FMModel(filepath=tmp_mdu_path)
```

Comment:
why testing the FMModel class, when thes `startdatetime`, `stopdatetime` are attributes of the `Time` class ?